toolkit/tools/imageconfigvalidator/imageconfigvalidator.go

```diff
@@ -54,5 +59,13 @@ func main() {
 // ValidateConfiguration will run sanity checks on a configuration structure
 func ValidateConfiguration(config configuration.Config) (err error) {
 	err = config.IsValid()
+	if err != nil {
+		return
+	}
+	err = validatePackages(config)
+	return
+}
+
+func validatePackages(config configuration.Config) (err error) {
 	return
 }
```

Comment:
Is this just preparation for later? Seems a bit pointless here.

Reply:
Yes. In hindsight this probably didn't need to be its own PR, but it made it once since is a new feature of the build system.

I add some checks later along with the verity packages.

---
In reply to: [549907978](https://github.com/microsoft/CBL-Mariner/pull/480#discussion_r549907978) [](ancestors = 549907978)